cves/kernel/CVE-2016-9754.yml

```diff
@@ -280,8 +280,8 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: 'The exploit involves user input to modify a file controlling buffer size.'
```

Comment:
How did you come to this conclusion?

This seems to be a vulnerability in a kernel data structure.

Reply:
Yes, but the exploit noted in [the fix commit](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=59643d1535eb220668692a5359de22545af579f6) involves editing a file that controls the buffer size used in the ring buffer, `/sys/kernel/debug/tracing/buffer_size_kb`. This is also noted in the vulnerability description on [NVD](https://nvd.nist.gov/vuln/detail/CVE-2016-9754).